src/Home.tsx

```diff
@@ -22,10 +22,10 @@ export default class Home extends React.Component {
           </Col>
         </Row>
         <Row>
-          <Col>
+          <Col lg={6} md={12}>
```

Comment:
I've been looking at the documentation for `Row` and `Col` elements:

Row: https://react-bootstrap.netlify.app/layout/grid/#row-props
Col: https://react-bootstrap.netlify.app/layout/grid/#col-props

It seems when you set these attributes on a `Col`, you're specifying how many columns of the grid the column spans. That requires foreknowledge of how many columns are _in_ the grid. And it also needs to be set on every column (as you've done here).

Setting `xs`, `sm`, `md`, etc. on the `Row` element instead seems to specify how many rows may appear side by side at each breakpoint. That seems to me to be slightly more straightforward and intuitive to understand.

In other worse, this might be better:

```
<Row xs={1} sm={1} md={1} lg={2} xl={2} xxl={2}>
```

Reply:
@krancour I agree. Setting breakpoints in the `Row` seems more straightforward. I'll do the required changes and send another commit!